portal-web/test/functional/com/liferay/portalweb/tests/mytest/myTestFile.testcase

```diff
@@ -0,0 +1,124 @@
+     definition {
+
+	property osgi.module.configuration.file.names = "com.liferay.frontend.js.loader.modules.extender.internal.configuration.Details.config";
+	property osgi.module.configurations = "waitTimeout=\"15\"";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Forms";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			Navigator.openURL();
+
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Forms");
+
+			FormsAdmin.teardownForms();
+
+			Navigator.gotoNavItem(navItem = "Element Sets");
+
+			FormsAdmin.teardownElementSets();
+
+			PagesAdmin.tearDownCP();
+		}
+	}
+
+
+     
+     @priority = 5
+    test addFormLiferay {
+        task("Criar o form") {
+            FormsAdmin.addForm();
+    }
+        task("editar o titulo") {
+            Form.editName(formName= "Form Liferay");
+        }
+        task("adicionar campo de texto") {
+            Form.addTextField();
+        }
+        task("mudar texto do campo de texto") {
+            Form.editFieldLabelMetal(fieldValue= "Full Name");
+            Form.save();
+        }
+        task("Adicionando field numeric") {
+           Form.gotoAddField(
+            fieldPositionNumber = 1,
+            fieldType = "Numeric");
+            Form.editFieldLabelMetal(fieldValue= "Age");
+            Form.save();
```

Comment:
Can you explain to me the reason for using so many` Form.save();`? This delays and hinders the performance of the C.I. 
Use only when really necessary.